ledgerblue/runScript.py

```diff
@@ -64,13 +66,20 @@ def decryptAES(self, data):
 			return self.loader.scpUnwrap(data)
 
 	dongle = getDongle(args.apdu)
+
+	targetId = args.targetId
+
+	if args.elfFile:
+		with get_elf_file(args.elfFile) as elf:
```

Comment:
It would make sense to hide this two lines under a direct call, something like `targetID = get_target_from_elf(args.elfFile)`. There's is no need to expose such complexity here.

Reply:
Done.